bpf/flows.c

```diff
@@ -69,6 +69,17 @@ struct {
     __uint(max_entries, 1 << 24);
 } direct_flows SEC(".maps");
 
+
+// Perf Buffer to submit packet payloads to userspace
+struct {
+    
+    __uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
+	__uint(key_size, sizeof(__u32));
+	__uint(value_size, sizeof(__u32));
+    __uint(max_entries, 80);
```

Comment:
just 80 entries ? is this experimental value ?

Reply:
This is an experimental value. Reverting.

Reply:
Fixed